implementations/rust/ockam/ockam_command/src/terminal/fmt.rs

```diff
@@ -0,0 +1,39 @@
+#[macro_export]
+macro_rules! fmt_ok {
+    ($input:expr) => {
+        format!("{} {}", "✔".light_green(), format!($input))
+    };
+    ($input:expr, $($args:expr),+) => {
+        format!("{} {}", "✔".light_green(), format!($input, $($args),+))
+    };
+}
+
+#[macro_export]
+macro_rules! fmt_info {
+    ($input:expr) => {
+        format!("{} {}", ">".light_green(), format!($input))
```

Comment:
We can probably work more on the colors we choose in the future.

I do think light green should be for `fmt_ok` and there should be a completely different color for `fmt_info`

Reply:
Yeah agree, I just took it from the output we currently have in the `ockam enroll` command. The good thing is that we'll only have to change it here.

Reply:
The most recent Clap release has some interesting theming stuff included behind an unstable flag - maybe we can offer them feedback or adopt it in the future. Docs are quite sparse as of today.

https://github.com/clap-rs/clap/issues/3234
Dependabot PR for us: https://github.com/build-trust/ockam/pull/4748

Reply:
That's cool! We could leverage that feature to integrate the clap native errors (https://docs.rs/clap_builder/4.2.3/src/clap_builder/builder/styled_str.rs.html#233-242).